commit 1be5f873e4bfb4502e9bea2a4e8519b278922073 (refs/changes/23/3023/1)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2017-12-01T02:13:59-08:00 (1 year, 10 months ago)
    
    CNAPI-728 Error when updating Nic tags of new east-3b servers

diff --git a/lib/endpoints/nics.js b/lib/endpoints/nics.js
index 518f389..ee2ada1 100644
--- a/lib/endpoints/nics.js
+++ b/lib/endpoints/nics.js
@@ -84,7 +84,6 @@ Nic.update = function handlerNicUpdate(req, res, next) {
     var wfParams = {
         nic_action: action,
         nics: req.params.nics,
-        server: server,
         server_uuid: uuid,
         target: uuid,
         origin: req.params.origin,
diff --git a/lib/workflows/server-update-nics.js b/lib/workflows/server-update-nics.js
index 96546bc..f136d75 100644
--- a/lib/workflows/server-update-nics.js
+++ b/lib/workflows/server-update-nics.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright (c) 2017, Joyent, Inc.
  */
 
 /*
@@ -20,7 +20,7 @@
  * - Workflow refreshes sysinfo to get the updated tags
  */
 
-var VERSION = '1.0.0';
+var VERSION = '1.0.1';
 
 var cnapiCommon = require('wf-shared').cnapi;
 var napiCommon = require('wf-shared').napi;
@@ -32,11 +32,6 @@ function validateParams(job, callback) {
         return;
     }
 
-    if (!job.params.server) {
-        callback(new Error('Must specify server'));
-        return;
-    }
-
     callback(null, 'All parameters OK!');
 }
 
